### PR TITLE
Allow App Preview to perform updates, attempt to reload session

### PR DESCRIPTION
**Pull request by wpride**
cross-request https://github.com/dimagi/formplayer/pull/193

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 1]
```diff
@@ -393,16 +399,19 @@ FormplayerFrontend.on('refreshApplication', function(appId) {
     if (!appId) {
         throw new Error('Attempt to refresh application for null appId');
     }
+    var sessionId = FormplayerFrontend.request('getCurrentSessionId');
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: what happens when someone is on the main screen of app preview and hits update and sessionId is null?

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 2]
```diff
@@ -412,7 +421,12 @@ FormplayerFrontend.on('refreshApplication', function(appId) {
         tfLoadingComplete(true);
     }).done(function() {
         tfLoadingComplete();
-        FormplayerFrontend.trigger('navigateHome');
+        if (typeof resp.responseJSON.tree !== 'undefined') {
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: this is kind of an awkward check. can just use: http://underscorejs.org/#isUndefined

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js (hunk 1 above)
Comment (wpride) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: In that case the sessionId will be null on Formplayer too and we'll just do a normal update with no attempt at replaying anything

**Review by benrudolph — APPROVED** [review]
sweet 🍬

**Pull request merged**